documentation/puppetdb_connection.markdown

```diff
@@ -64,6 +64,21 @@ This setting can let the Puppet master stay partially available during a PuppetD
 
 The default value is false.
 
+## Differences in Puppet Enterprise
+
+### `include_unchanged_resources`
```

Comment:
I'm not sure, I went back and forth on that, which is why I made PE notes, I think we want both a "All the PE things" doc and we want pe features to be in places where a user would expect to find them (like in the reports doc).